controllers/datafoundation.go

```diff
@@ -114,3 +119,29 @@ func (r *dataFoundationReconciler) reconcileOnboardingValidationSecret() error {
 	}
 	return nil
 }
+
+func (r *dataFoundationReconciler) reconcileRookCephOperatorConfig() error {
+	rookConfigMap := &corev1.ConfigMap{}
+	rookConfigMap.Name = "rook-ceph-operator-config"
+	rookConfigMap.Namespace = r.managedFusionOffering.Namespace
```

Comment:
Why do we have some reconciled resources defined on the reconciler (and identified in `initReconciler`) and some defined  in the reconciler function?

Reply:
Updated